packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -923,14 +923,24 @@ export class PHP implements Disposable {
 		return FSHelpers.isDir(this[__private__dont__use].FS, path);
 	}
 
-	/**
-	 * Checks if a file (or a directory) exists in the PHP filesystem.
-	 *
-	 * @param  path - The file path to check.
-	 * @returns True if the file exists, false otherwise.
-	 */
-	fileExists(path: string) {
-		return FSHelpers.fileExists(this[__private__dont__use].FS, path);
+	/** @inheritDoc */
+	isFile(path: string): boolean {
+		if (!this.fileExists(path)) {
+			return false;
+		}
+		return this[__private__dont__use].FS.isFile(
```

Comment:
Let's move this logic to FSHelpers in the same way as for all other methods in the `PHP`.

Reply:
> Let's move this logic to FSHelpers in the same way as for all other methods in the PHP.

Done!